ng/src/web/components/folding/folding.js

```diff
@@ -107,12 +116,16 @@ export const withFolding = (Component, defaults = {}) => {
     }
 
     return (
-      <Component
-        style={new_style}
-        {...props}
-        onTransitionEnd={onFoldStepEnd}
-        onAnimationEnd={onFoldStepEnd}
-      />
+      <Div
+        {...new_style}
+      >
+        <Component
+          style={new_style}
```

Comment:
This prop should be removed from Component. It's being applied to the Div already